Fixed #31928 -- Fixed detecting an async get_response in various middlewares. (#13344)
django/django: docs/releases/3.1.1.txt

  ``CommonPasswordValidator`` and ``settings.py`` generated by the
  :djadmin:`startproject` command, when user didn't have permissions to all
  intermediate directories in a Django installation path (:ticket:`31912`).

* Fixed detecting when the wrapped ``get_response`` callable is async in

>   various builtin middlewares (:ticket:`31928`).

Reviewer: @carltongibson Can you take a look at the new version of release notes?

> 
> * Fixed detecting when the wrapped ``get_response`` callable is async in
>   various builtin middlewares (:ticket:`31928`).

Reviewer 2: It's fine. Maybe: 

```suggestion

* Fixed detecting an async ``get_response`` callable in various builtin 
  middlewares (:ticket:`31928`).
```

Thanks!
